CHANGES.md

```diff
@@ -26,6 +26,8 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 
 * Expose Public IP (DCOS_OSS-4514)
 
+* Expose mesos flag to persist network cni root directory (DCOS_OSS-4667) 
```

Comment:
I'd say "Expose a Mesos flag to allow the network CNI root directory to be persisted across host reboot"

Is there a place to tell the user that changing this flag requires a reboot or drain?

Reply:
will change the description. I guess that would be part of the documentation where we specify this new flag